daemon/network.go

```diff
@@ -45,7 +45,9 @@ func (daemon *Daemon) FindNetwork(idName string) (libnetwork.Network, error) {
 	// 3. match by ID prefix
 	list := daemon.GetNetworksByIDPrefix(idName)
 	if len(list) == 0 {
-		return nil, errors.WithStack(networkNotFound(idName))
+		// The Error type ErrNoSuchNetwork is also used by the controller to try to create a managed network
+		// asking to the manager, for this reason the type cannot be changed
+		return nil, errors.WithStack(libnetwork.ErrNoSuchNetwork(idName))
```

Comment:
looks like `networkNotFound()` is no longer used now.

Wondering can we wrap the `libnetwork.ErrNoSuchNetwork(idName)` so that it's still detected as an `libnetwork.ErrNoSuchNetwork()` error, but also implements `NotFound()` to make the API translations work?